### PR TITLE
Bocm 19 migrate all code to esm

### PR DESCRIPTION
**Pull request by radum**

### CODE REVIEW

File: apps/web/src/server/app/app.express.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+import bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import serveStatic from 'serve-static';
+import helmet from 'helmet';
+import { __dirname } from '../lib/helpers.js';
+import { routes } from './routes.js';
+import { config } from '../config/config.js';
+import stripQueryParametersDevelopment from '../lib/nunjucks-filters/strip-query-parameters.js';
+// import resourceCSS from '../_data/resourceCSS.json' assert { type: 'json' };
+// import resourceJS from '../_data/resourceJS.json' assert { type: 'json' };
```
Comment (mayya-bond-kc) [inline apps/web/src/server/app/app.express.js:18]: Do we need this? If so, leave a comment why

**Review by mayya-bond-kc — APPROVED** [review]
lgtm 👍

**Review by MariaValeroGonzalezKC — APPROVED** [review]

**Comment by MariaValeroGonzalezKC** [comment]
👍

**Pull request merged**